app/controllers/admin/schools/participants_controller.rb

```diff
@@ -4,19 +4,18 @@ module Admin
   class Schools::ParticipantsController < Admin::BaseController
     skip_after_action :verify_authorized
     skip_after_action :verify_policy_scoped
-    before_action :set_school
 
     def index
-      @participants = @school.active_ecf_participants.order(:full_name).includes(
-        early_career_teacher_profile: [:mentor, school_cohort: %i[school cohort]],
-        mentor_profile: [school_cohort: %i[school cohort]],
-      )
+      @participant_profiles = policy_scope(ParticipantProfile::Ecf, policy_scope_class: ParticipantProfilePolicy::Scope)
+        .where(school_cohort_id: SchoolCohort.where(school: school).select(:id))
```

Comment:
`ParticipantProfile::Ecf` has a school on it, so maybe we could write `where(school: school)`? Or `where(schools: school` depending on how mean rails feels today? :P

Reply:
Interestingly enough - tried this but it didn't work as rails tried to filter on `school` column which does not exist...